src/server.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import registerApiDocs from './swagger';
+import auth from './routes/auth';
+
+const app = express();
+
+app.use(express.urlencoded({ extended: false }));
```

Comment:
This should not be duplicated. Since you have instantiated express in `app.js`, you can import it here instead.